tasks/go.py

```diff
@@ -436,7 +436,7 @@ def tidy(ctx):
 @task
 def check_go_version(ctx):
     go_version_output = ctx.run('go version')
-    # result is like "go version go1.21.11 linux/amd64"
+    # result is like "go version go1.22.4 linux/amd64"
```

Comment:
I'm not sure we need to change this comment

Reply:
Yeah we clearly don't need to, but it makes the output format of `go version` clearer, and doesn't cost much anyway 😄